index.php

```diff
@@ -0,0 +1,53 @@
+<html lang="en-GB">
+	<head>
+		<title>Cocktails</title>
+		<link rel="stylesheet" type="text/css" href="normalize.css" />
+		<link rel="stylesheet" type="text/css" href="style.css" />
+		<link href="https://fonts.googleapis.com/css?family=Mrs+Sheppards|Sen&display=swap" rel="stylesheet">
+	</head>
+
+	<body>
+		<!-- top banner -->
+		<header>
+			<div class="container_top">
+				<h1>Cocktails</h1>
+			</div>
+		</header>
+
+		<div class="container_main">
+			<div class="container_link">
+                <!-- Link goes to the "add cocktail" page -->
+			<a href="./add_item.php">Add new cocktail</a>
+			</div>
+
+			<div class="cocktails_box">
+
+                <?php
+
+                // Accessing DB from PHP
+                $db = new PDO('mysql:host=db; dbname=Cocktails', 'root', 'password');
+                $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+                $query=$db->prepare("SELECT* FROM `cocktails`;");
```

Comment:
SELECT * isn't good practice. I'd recommend that you SELECT only the columns you need to populate your page

Reply:
$query=$db->prepare("SELECT `name`, `alcohol_base`, `taste_profile`, `ingredients`, `method`, `strength`, `served`  FROM `cocktails`;");

Reply:
changed to this